src/rgx/block.js

```diff
@@ -10,33 +10,69 @@
  * governing permissions and limitations under the License.
  */
 module.exports = [
+  /_debugbar/,
   /.+\.axd/,
+  /.+\.log$/,
+  /(?!\/\.well-known)\/\./,
+  /(Homestead|circle|_?config)\.ya?ml$/,
   /[()'"]/, // braces and escapes
   /[Tt]hank.*\.txt$/,
+  /\.\./, // do not go a directory up
+  /\.gitignore/,
+  /\.htaccess/,
+  /\.htpasswd/,
+  /\.swf$/,
+  /\/_?debug/,
   /\/(.?php.*|tiki.*|test.*|user.*|wp-|ws|t.*|v.*|u.*|wiz.*|[Ww]eb)([aA]dmin|wiz.*)/,
+  /\/(bower|config|\.[-_\w]+|app|\w+404)\.json/, // a bunch of json files, leftover from development
+  /\/(plugins?)\/.*(servlet|metrics)/,
+  /\/(solr|jira|confluence|jenkins|servicedesk|app)\/.*(config|plugins|user|kibana)/,
   /\/[Ww][Ss]_/,
   /\/[Ww]ord[Pp]ress|\/WebSphere|\/wwwroot/,
+  /\/@/,
+  /\/\??%2[AaEe]/, // escaped . and *
+  /\/\.\./,
+  /\/\.env/,
   /\/\//, // double slashes
   /\/\/etc\//,
-  /\/\.env/,
+  /\/\$%7[bB]/,
+  /\/%24%7[bB]/,
+  /\/%252e/,
+  /\/%C2%B0/,
+  /\/App_Data\//,
+  /\/autoconfig/,
+  /\/cloudfoundryapplication/,
+  /\/CVS\/Entries/,
   /\/etc\/passwd/,
   /\/fckeditor/,
+  /\/ipython\/tree/,
+  /\/irj\/go\//,
+  /\/jkstatus$/,
+  /\/jolokia/,
+  /\/logfile/,
+  /\/nginx.*conf$/,
   /\/tiny_?mce/,
   /\/umbraco/,
+  /\/v[1-9]\/_catalog/,
   /\/wp-(app|admin|config|contents?|includes|login|rss.*|register)/,
   /\/zimbra/,
-  /\.\./, // do not go a directory up
-  /\.swf$/,
-  /\/\.\./,
   /%(2[26]|3([aA]|[dD])|5[cC]|7[cC])/, // note that the % is an encoded percent sign
+  /%0[a-fA-F0-9]/,
+  /%23/, // %23 = #
```

Comment:
interesting....this means, the requests are not sent by a browser....

Reply:
No, they are sent by kids with scripts :)